### PR TITLE
Set RTT to streams. RTT is only checked once for now.

### PR DESCRIPTION
**Pull request by harryz2000**

### CODE REVIEW

File: src/rtmp/rtmpconnection.cpp [hunk 1]
```diff
@@ -844,12 +844,21 @@ void RTMPConnection::ProcessControlMessage(DWORD streamId,BYTE type,RTMPObject*
...
 					break;
 				case RTMPUserControlMessage::PingResponse:
+				{
 					//Get response
 					DWORD ping = event->GetEventData();
 					//Get roundtrip delay
 					rtt = getDifTime(&startTime)/1000-ping;
+					
+					//Check if a stream has been created with that id
+					pthread_mutex_lock(&mutex);
```
Comment (murillo128) [inline src/rtmp/rtmpconnection.cpp:854]: can we use range for loops instead?

**Review by murillo128 — CHANGES_REQUESTED** [review]
minor change required, lgtm otherwise

File: src/rtmp/rtmpconnection.cpp (hunk 1 above)
Comment (harryz2000) [reply src/rtmp/rtmpconnection.cpp:854]: Yes. Done

**Review by murillo128 — APPROVED** [review]
lgtm

**Pull request merged**